services/hydra/default.nix

```diff
@@ -33,18 +28,6 @@ in
       "https://github.com/NixOS/"
     ];
 
-    nixpkgs.config = {
```

Comment:
@zimbatm nixpkgs.config was added in 19581331ed38654514ee14f09a5c19123da7da8e, I don't think we need it now?

Reply:
I don't know if those still apply now that we use `nixpkgs.lib.nixosSystem`? I think it's using the passed instance of nixpkgs instead.

Reply:
Sorry, I meant that I don't think that we're still running builds for anything that requires them?

Reply:
I propose merging this and find out :)